### DISCUSSION TITLE
[Kavita] Kavita Joins the Fediverse

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2553] [Ideas]
### Idea Description

Posted by aschrijver
· Nov 2021
· 14 Upvotes
https://feats.kavitareader.com/posts/13/kavita-joins-the-fediverse

Description

It looks like Kavita is ideally suited to add federation support via ActivityPub protocol, so that self-hosted instances start to federate together. I'll allow different people to share resources, follow each other, create multiple communities and groups, and comment, rate, review and do other social interactions on the collections of their friends.

There's a number of somewhat similar Fediverse applications already and multiple ways you might facilitate cool integrations to other federated apps, such as Mastodon, Pixelfed, Lemmy and Peertube.

Check https://fediverse.party/ and https://git.feneas.org/feneas/fediverse/-/wikis/watchlist-for-activitypub-apps to explore other apps.

PS. The fediverse is FULL of Manga fans, btw :D

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
Going to officially close this out. I'm not interested in this FR.
